etc/scripts/build-stack-installer.hs

```diff
@@ -1,5 +1,5 @@
 {- stack script
-    --resolver lts-14.27
+    --resolver lts-19.7
```

Comment:
Should the scripts also be moved on to a GHC 9.2.3 resolver? Otherwise people will need to maintain two versions of GHC.

Reply:
Yeah, good point. I will make the changes here and the other script.